src/flamegraph/color/palettes.rs

```diff
@@ -59,6 +59,33 @@ pub(super) mod perl {
     }
 }
 
+pub(super) mod python {
+    use crate::flamegraph::color::BasicPalette;
+    use std::path::Path;
+
+    pub fn resolve(name: &str) -> BasicPalette {
+        if name.starts_with("native@") {
+            // austin-specific format for native calls
+            return BasicPalette::Aqua;
+        } else if Path::new(name).iter().any(|c| c == "site-packages") {
+            return BasicPalette::Yellow;
+        } else if Path::new(name).iter().any(|c| {
+            c.to_str()
+                .unwrap_or("")
+                .to_ascii_lowercase()
+                .strip_prefix("python")
```

Comment:
This allocates, which I'd like to avoid. I think you can use this instead:
```rust
c.strip_prefix("python").or_else(|| c.strip_prefix("Python"))
```